kubernetes/samples/scripts/create-soa-domain/common/createSOADomain.py

```diff
@@ -0,0 +1,605 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```

Comment:
Did this script originally come from somewhere else, or is it new?   If it's a copy, I wonder if it should be downloaded as part of the sample.

Reply:
The reason I ask is that it looks complex enough that I suspect it'll change over time, or be different for different versions of the SOA stack.

Reply:
The base reason for my question remains - the concern is future maintenance:   Is this the sole repository for this script, in which case we can assume we're the owner, or is it a copy from some external repository with a different owner/maintainer?  If it's the latter, the sample should download it (probably a specific version of it with some sort of option to specify different versions) instead of copy it here.